src/DocumentFormat.OpenXml/OpenXmlAttribute.cs

```diff
@@ -13,6 +13,8 @@ namespace DocumentFormat.OpenXml
     /// </summary>
     public struct OpenXmlAttribute : IEquatable<OpenXmlAttribute>
```

Comment:
Is it possible marks as ReadOnly in future?

Reply:
Not that I can think of. I'm also not certain there's much of a value - adding `readonly` as a struct isn't breaking, so it's something we can do once the setters are removed (see https://blog.paranoidcoding.com/2019/03/27/readonly-struct-breaking-change.html)